website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -86,15 +87,15 @@ Some caption tools work for virtual events only, others for in-person and virtua
 
 Captions at the bottom of the screen are not visible unless attendees are sitting at the front. Consider adding them at the top. If that isn't feasible,  you might want to place the screen higher to ensure people in the back can also read them. 
 
-Some captions are highly **accurate but lag** significantly. That creates a disconnect between the message received and the speaker/audience (not syncing with body language, laughter, etc.), which can create a feeling of ‘missing out’
+Some captions are highly **accurate but lag** significantly. That creates a disconnect between the message received and the speaker/audience (not syncing with body language, laughter, etc.), which can create a feeling of ‘missing out.’
 
 Unlike hearing people, deaf/hoh individuals can only focus on the captions or look at the speaker/slides (e.g., a technical graphic). So if they do look away, they will miss what's said. Hence, leaving content on the screen longer by **adding more than one line** will help people rely on captions. 
 
 Every session should have captions in the main conference language **on the screen by default** (vs. captions available on the phone). Reading captions on-screen makes attendees feel more engaged than looking at their phones (speakers might think they aren't interested). With a phone, attendees are constantly required to shift focus between their phones and speaker(s) and will miss parts of the message. This is even more critical when there are live demos because, to understand what is going on, attendees must see what's on-screen and read captions at the same time. Captions on the phone make that impossible. 
 
 #### Caption use cases beyond deaf/hoh
 
-Some attendees can hear but may be on the autism spectrum and have a hard time processing auditory cues — captioning is incredibly beneficial for them, too. Additionally, non-native speakers who aren’t fluent can benefit from having words on the screen.
+Some attendees can hear but may be on the autism spectrum and have a hard time processing auditory cues — captioning is incredibly beneficial for them, too. Additionally, non-native speakers who aren’t fluent will benefit from captions as well. 
```

Comment:
Some attendees can hear but may be neurodivergent ( e.g. on the autism spectrum, with ADHD, and more) and have a hard time processing auditory cues — captioning is incredibly beneficial for them, too. Additionally, non-native speakers who aren’t fluent will benefit from captions as well.

Reply:
Updated